docs/components/connectors/out-of-the-box-connectors/slack-inbound.md

```diff
@@ -0,0 +1,205 @@
+---
+id: slack-inbound
+title: Slack Inbound Connector
+sidebar_label: Slack Inbound Connector
+description: Trigger a Slack bot to start a BPMN process with a bot event or a slash command
+---
+
+The **Slack Inbound Connector** is a Connector that allows you to start or continue
+a BPMN process triggered by a [Slack](https://slack.com/) message.
+
+## Create a Slack Inbound Connector task
+
+1. Start building your BPMN diagram. You can use the **Slack Inbound Connector** with either a **Start Event** or **Intermediate Catch Event**.
+2. Select the applicable element and change its template to an **Slack Inbound Connector**.
+3. Fill in all required properties.
+4. Complete your BPMN diagram.
+5. Deploy the diagram to activate the webhook.
+6. Navigate to the **Webhooks** tab in the properties panel to see the webhook URL.
+
+## Make your Slack Inbound Connector for receiving notifications executable
+
+![Start event SNS connector](../img/connectors-slack-inbound-start-filled.png)
+
+![Intermediate catch event SNS connector](../img/connectors-slack-inbound-intermediate-filled.png)
+
+1. In the **Webhook Configuration** section, configure the **Webhook ID**. By default, **Webhook ID** is pre-filled with a random value. This value will be a part of the Slack event subscription or slash command URL.
```

Comment:
does slack have a fitting documentation for this step, if so, it would be great if we can link it.

Reply:
Yes, I cross-referenced this completely in the section `Wiring with Slack`